app/assets/stylesheets/arclight/modules/search_results.scss

```diff
@@ -149,12 +149,28 @@ article.document {
   fill: $gray-600;
 }
 
-.btn-outline-secondary:not(:disabled):not(.disabled).active svg, .btn-outline-secondary:hover svg {fill: #fff;}
-
-.btn-outline-secondary:not(:disabled):not(.disabled).active, .show > .btn-outline-secondary.dropdown-toggle {
```

Comment:
Were these styles intentionally dropped, or just the result of a bad merge conflict?

Reply:
That was intentional. The first one I just reformatted. The second was a duplicate style I remember from when I paired with Gordon and I forget that we didn't delete it.

Reply:
Just as a helpful heads up and reminder, when this https://github.com/projectblacklight/blacklight/pull/2134 is shipped and released, we can remove these.